src/Resources/config/routing/admin.yaml

```diff
@@ -2,7 +2,7 @@ setono_sylius_analytics_admin_property:
     resource: |
         section: admin
         alias: setono_sylius_analytics.property
-        templates: SyliusAdminBundle:Crud
+        templates: "@SyliusAdmin\\Crud"
```

Comment:
Isn't the correct syntax `@SyliusAdmin/Crud`, @Roshyo?

Reply:
I followed the example of Sylius' official documentation:
https://docs.sylius.com/en/1.9/cookbook/entities/custom-model.html#define-routing-for-entity-administration

Reply:
Okay, I found related PRs/issues:

- https://github.com/Sylius/Sylius/pull/9726
- https://github.com/symfony/symfony/pull/39142

It seems it should work in these SF versions:

- v4.4.18
- v5.1.10
- v5.2.1

Can you test it out? I would like to use the 'correct' syntax. Doesn't that make sense? Wdyt?

Reply:
I have just tested and both solutions(`"@SyliusAdmin\\Crud"` and `"@SyliusAdmin/Crud"`) works in Symfony v4.4.18 and Symfony v5.2.1. In my opinion it's hard to say which syntax is more correct.